source/tinyengine_debug_utilities.h

```diff
@@ -0,0 +1,13 @@
+#include <stdio.h>
+#include <windows.h>
+
+static void
+Print(char* Format, ...)
```

Comment:
PrintDebugString instead since you can only see these string if you are running in VS